scripts/abi_artifact_generator/templates/printers.inc.jinja_template

```diff
@@ -0,0 +1,199 @@
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
+#*/
+
+/*(- generation_notice )*/
```

Comment:
This thread is a reminder about a carry-over thread:
https://github.com/revng/revng-qa/pull/17#discussion_r1354321365

Reply:
> AFAIU this decodes a global buffer containing data dumped by the various functions. `printers.c` does not convey this very well. Can we give a proper name to the buffer above and call all of this `decode_$NAME.c`?

Done. Also, I split buffer up, now it's `value_dumps`, `size_dumps` and `address_dumps`. What do you think?